### PR TITLE
cloud: add release notes

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 1]
```diff
@@ -8,6 +8,40 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
...
+
+**Console changes**
+
+- Update the [Monitoring](/tidb-cloud/built-in-monitoring.md#view-the-monitoring-page) page for [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) clusters to use [node-level resource metrics](/tidb-loud/built-in-monitoring.md#server).
+
+    With node-level resource metrics, you can see a more accurate representation of resource consumption to better understand the actual usage of purchased services.
+
+    To access these metrics, navigate to the [Monitoring](/tidb-cloud/built-in-monitoring.md#view-the-monitoring-page) page of your cluster, and then check the **Server** category under the **Metrics** tab.
+
+- Optimize the [Billing](/tidb-cloud/tidb-cloud-billing.md#billing-details) page by reorganizing the billing items in **Summary by Project** and **Summary by Service**, which makes the billing information more clear.
```
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-release-notes.md:43]:
```suggestion
- Optimize the [Billing](/tidb-cloud/tidb-cloud-billing.md#billing-details) page by reorganizing the billing items in **Summary by Project** and **Summary by Service**, which makes the billing information clearer.
```

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 2]
```diff
@@ -8,6 +8,40 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
...
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2023.
 
+## April 11, 2023
+
+**General changes**
+
+- Improve the load balance of TiDB and reduce connection drops when you scale TiDB nodes of [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) clusters hosted on AWS.
+
+    - Support automatically migrating existing connections to new TiDB nodes when you scale out TiDB nodes.
+    - Support automatically migrating connections to available TiDB nodes when you scale in TiDB nodes.
```
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-release-notes.md:18]:
```suggestion
    - Support automatically migrating existing connections to available TiDB nodes when you scale in TiDB nodes.
```

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 3]
```diff
@@ -8,6 +8,40 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
...
+
+  Currently, this feature is only provided for Dedicated Tier clusters that are hosted in the AWS `Oregon (us-west-2)` region.
+
+- Support the [New Relic](https://newrelic.com/) integration.
+
+    With the New Relic integration, you can configure TiDB Cloud to send metric data of your TiDB clusters to [New Relic](https://newrelic.com/). Then, you can monitor and analyze both your application performance and your TiDB database performance on [New Relic](https://newrelic.com/). This feature can help you quickly identify and troubleshoot any potential issues that might arise and reduce the resolution time.
```
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-release-notes.md:24]:
```suggestion
    With the New Relic integration, you can configure TiDB Cloud to send metric data of your TiDB clusters to [New Relic](https://newrelic.com/). Then, you can monitor and analyze both your application performance and your TiDB database performance on [New Relic](https://newrelic.com/). This feature can help you quickly identify and troubleshoot potential issues and reduce the resolution time.
```

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 4]
```diff
@@ -8,6 +8,40 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
...
+
+    For integration steps and available metrics, see [Integrate TiDB Cloud with New Relic](/tidb-cloud/monitor-new-relic-integration.md).
+
+- Add the following [changefeed](/tidb-cloud/changefeed-overview.md) metrics to the Prometheus integration for Dedicated Tier clusters.
+
+    - `tidbcloud_changefeed_latency`
+    - `tidbcloud_changefeed_replica_rows`
+
+      If you have [integrated TiDB Cloud with Prometheus](/tidb-cloud/monitor-prometheus-and-grafana-integration.md), you can monitor the performance and health of changefeeds in real time using these metrics. Additionally, you can easily create alerts to monitor the metrics using Prometheus.
```
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-release-notes.md:33]:
```suggestion
    If you have [integrated TiDB Cloud with Prometheus](/tidb-cloud/monitor-prometheus-and-grafana-integration.md), you can monitor the performance and health of changefeeds in real time using these metrics. Additionally, you can easily create alerts to monitor the metrics using Prometheus.
```

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 5]
```diff
@@ -8,6 +8,40 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
...
+## April 11, 2023
+
+**General changes**
+
+- Improve the load balance of TiDB and reduce connection drops when you scale TiDB nodes of [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) clusters hosted on AWS.
+
+    - Support automatically migrating existing connections to new TiDB nodes when you scale out TiDB nodes.
+    - Support automatically migrating connections to available TiDB nodes when you scale in TiDB nodes.
+
+  Currently, this feature is only provided for Dedicated Tier clusters that are hosted in the AWS `Oregon (us-west-2)` region.
```
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-release-notes.md:20]:
```suggestion
  Currently, this feature is only provided for Dedicated Tier clusters that are hosted on the AWS `Oregon (us-west-2)` region.
```

File: tidb-cloud/tidb-cloud-release-notes.md (hunk 4 above)
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-release-notes.md:33]:
```suggestion
  If you have [integrated TiDB Cloud with Prometheus](/tidb-cloud/monitor-prometheus-and-grafana-integration.md), you can monitor the performance and health of changefeeds in real time using these metrics. Additionally, you can easily create alerts to monitor the metrics using Prometheus.
```

Comment (hfxsd) [reply tidb-cloud/tidb-cloud-release-notes.md:33]: duplicate

**Review by ran-huang — APPROVED** [review]

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 6]
```diff
@@ -8,6 +8,40 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
...
+**General changes**
+
+- Improve the load balance of TiDB and reduce connection drops when you scale TiDB nodes of [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) clusters hosted on AWS.
+
+    - Support automatically migrating existing connections to new TiDB nodes when you scale out TiDB nodes.
+    - Support automatically migrating existing connections to available TiDB nodes when you scale in TiDB nodes.
+
+  Currently, this feature is only provided for Dedicated Tier clusters that are hosted on the AWS `Oregon (us-west-2)` region.
+
+- Support the [New Relic](https://newrelic.com/) integration.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-release-notes.md:22]:
```suggestion
- Support the [New Relic](https://newrelic.com/) integration for [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) clusters.
```

**Pull request merged**